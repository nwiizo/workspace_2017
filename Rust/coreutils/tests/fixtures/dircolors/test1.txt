.xspf 00;36
NORMAL 00 # no color code at all
FILE 00 # regular file: use no color at all
# Below are the color init strings for the basic file types. A color init
# string consists of one or more of the following numeric codes:
# Attribute codes:
# 00=none 01=bold 04=underscore 05=blink 07=reverse 08=concealed
# Text color codes:
# 30=black 31=red 32=green 33=yellow 34=blue 35=magenta 36=cyan 37=white
# Background color codes:
# 40=black 41=red 42=green 43=yellow 44=blue 45=magenta 46=cyan 47=white
TERM screen*
.rar 12;34
.tar 01;31
.tgz 01;31
.arc 01;31
ORPHAN 40;31;01 # symlink to nonexistent file, or non-stat'able file ...
MISSING 00 # ... and the files they point to
.arj 01;31
.taz 01;31
TERM gnome
SETUID 37;41 # file that is setuid (u+s)
SETGID 30;43 # file that is setgid (g+s)
CAPABILITY 30;41 # file with capability
