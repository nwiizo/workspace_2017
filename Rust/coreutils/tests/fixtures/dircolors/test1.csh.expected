setenv LS_COLORS '*.xspf=00;36:no=00:fi=00:su=37;41:sg=30;43:ca=30;41:'
